src/main.c

```diff
@@ -118,11 +118,11 @@ int main(int argc, char** argv)
     register_hdmicec_hal_l1_tests ();
```

Comment:
Please register L1 cases similar to L2 here.  Differentiate between Source and sink and call the test suite registration in this file to maintain uniformity between L1 and L2 test cases.

Reply:
Moved the register functions to main.c. similar to l2 tests.

Reply:
renamed existing register function as register_hdmicec_hal_l1_common_tests and added register_hdmicec_hal_l1_sink_tests & register_hdmicec_hal_l1_source_tests for sink and source device respectively.